src/types/config.js

```diff
@@ -85,6 +85,8 @@
  *   Name of the base theme to use.
  * @prop {string} [usernameUrl]
  *   URL template for username links
+ * @prop {string} [assetRoot]
```

Comment:
I not 100% sure these 2 go here, I could use feedback on this

Reply:
I would leave these out. These properties may be set in the host config, but they are not used by the sidebar. This type only lists the host config properties which are used by the sidebar - see the comment above. `sidebarAppUrl` is used by "annotator" and `assetRoot` is used by the boot script.